shells/pipes-shell/node/node.js

```diff
@@ -19,7 +19,7 @@ import {version, paths, storage, test} from './config.js';
 
 // dependencies
 import {RamSlotComposer} from '../../lib/components/ram-slot-composer.js';
-import {initPipe} from '../pipe.js';
+import {initPipe, initArcs} from '../pipe.js';
```

Comment:
This change was made in `node.js` but don't we want to make it in `web.js`?

We are using pipe-shell in web, not node.

Reply:
@yuangu-google to confirm which code path makes sense (and if we're not using one, maybe we should delete/archive it?).

Reply:
I made the changes in both - node and web. 
I thought they are consistent, and wanted to keep them this way, shouldn't they be?

Reply:
Ah, I didn't see the web one at the bottom. `node` isn't being used currently, if it's not much work to keep it in sync that's desirable, sure. But if it turns out to be a hassle ongoing I think it'd be ok to remove it and we could pull it back out of the git archive if needed. Up to you.

Reply:
in this case it was trivial to keep them in sync, so i did.